sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -693,11 +693,11 @@ class FileBasedDataSourceSuite extends QueryTest
   }
 
   test("SPARK-22790,SPARK-27668: spark.sql.sources.compressionFactor takes effect") {
-    Seq(1.0, 0.5).foreach { compressionFactor =>
+    Seq(1.0, 0.4).foreach { compressionFactor =>
```

Comment:
why do we need to change this?

Reply:
https://issues.apache.org/jira/browse/ORC-984 starts to add a metadata about a ORC writer version name. That increases a few byte per file.